src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -351,7 +361,7 @@ def preprocess(self, inputs, chunk_length_s=0, stride_length_s=None, ignore_warn
                 processed["stride"] = stride
             yield {"is_last": True, **processed, **extra}
 
-    def _forward(self, model_inputs):
+    def _forward(self, model_inputs, **generate_kwargs):
```

Comment:
```suggestion
    def _forward(self, model_inputs, generate_kwargs=None):
        if generate_kwargs is None:
            generate_kwargs = {}
```

Reply:
@Narsil can we just define default arg `generate_kwargs={}`?

Reply:
No, it's discouraged, because of this : https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/